packages/core/test/chainbridge/beaconOracle.js

```diff
@@ -22,8 +23,13 @@ contract("BeaconOracle", async (accounts) => {
   const testRequestTime = 123;
   const testPrice = "6";
 
-  beforeEach(async function () {
+  before(async () => {
+    [owner] = accounts;
```

Comment:
This order seems off, shouldn't `accounts = await web3.eth.getAccounts()` precede `[owner] = accounts`?

Reply:
Yep! This is broken. Fixing this and will push soon.